style/article-form.css

```diff
@@ -19,9 +20,36 @@ a.c-learning-resource-form__abort-button.c-button {
   }
 }
 
+.c-agreement-form {
+  background-color: var(--brand-grey--lightest);
+  padding-bottom: 3rem;
+
+  &__header {
+    font-size: 1.2rem;
+    background-color: var(--brand-color--secondary);
+    color: white;
+  }
+}
+
+.c-audio-form,
+.c-image-form {
+  &__header {
+    background-color: var(--brand-color--dark);
+    color: white;
+  }
+}
+
+.c-topic-article-form,
+.c-learning-resource-form {
+  &__header {
+    background-color: var(--yellow-1);
+  }
+}
+
 .c-topic-article-form,
 .c-audio-form,
 .c-image-form,
+.c-agreement-form,
```

Comment:
Samme her. Dette må ryddes opp i snart

Reply:
Denne lar jeg ligge for nå. Driver å tenker på hvordan jeg best mulig kan gjøre dette pent.